docs/source/resources/package-spec.rst

```diff
@@ -95,15 +95,13 @@ conda package is composed of the first 3 values, as in:
 
    * - build_number
      - integer
-     - A non-negative integer representing the build number of
-       the package.
+     - A non-negative integer representing the build number of the package.
 
-       Unlike the build string, the ``build_number`` is inspected by
-       conda. Conda uses |br| it to sort packages that have otherwise
-       identical names and versions |br| to determine the latest one.
-       This is important because new builds that |br| contain bug
-       fixes for the way a package is built may be added to a
-       repository.
+       Unlike the build string, the ``build_number`` is inspected by conda.
+
+       Conda uses it to sort packages that have otherwise identical names and versions to determine the latest one.
```

Comment:
Are you wanting these to all be their own paragraphs? That's currently what will happen with putting all sentences on their own lines. You can just write the text like you would write it in a word processor.

Reply:
There be dragons here.  I don't want to write them as their own paragraphs BUT because this is in a table AND the double backticks, it renders a very long line.  This is why I inserted breaks so that it would render ok.  Doing it this way takes away the embedded breaks which is out of season and makes it render ok.